cinema/views.py

```diff
@@ -22,6 +25,7 @@
     MovieListSerializer,
     OrderSerializer,
     OrderListSerializer,
+    MovieImageSerializer,
```

Comment:
Ensure that the `MovieImageSerializer` is correctly imported and used in the `MovieViewSet` class, especially in the `upload_image` action. This is crucial for handling image uploads properly.